validator_test.sh

```diff
@@ -0,0 +1,176 @@
+args=("$@")
+## Set up envs
+echo "What network you going to interact:"
+echo "1. CosmosHub"
+echo "2. Juno"
+
+read -N 1 i
+echo $i
```

Comment:
This echoes back user input twice and doesn't wait until user presses Enter (making them scared).
I would rather use `read i` in this case.